hack/charts/cluster-api-operator/values.yaml

```diff
@@ -1,12 +1,30 @@
 ---
 # ---
 # Cluster API provider options
-core: ""
-bootstrap: ""
-controlPlane: ""
-infrastructure: ""
-ipam: ""
-addon: ""
```

Comment:
So, now options have been updated to use structured objects for providers instead of plain strings. Could it be potentially a breaking change since existing deployments rely on the old string format and need to be updated to the new format?

Reply:
It is, this would be a major upgrade for the helm chart.